src/refactor_nrepl/core.clj

```diff
@@ -376,3 +380,11 @@
   [symbol-or-keyword]
   (when (prefix symbol-or-keyword)
     symbol-or-keyword))
+
+(defmacro with-clojure-version->=
+  "Guard the evaluation of `body` with a test on the current clojure version."
+  {:style/indent 1}
+  [{:keys [major minor] :as _clojure-version} & body]
+  (when (and (>= (:major *clojure-version*) major)
+             (>= (:minor *clojure-version*) minor))
```

Comment:
I think that if `*clojure-version*` is 2.0, then the minor comparison can make the condition fail

IOW if major is > (but not >=), then minor is irrelevant?

Reply:
Fixed this, although I doubt we'll ever see a clojure 2.0 (would break their commitment to backward compatibility)